src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -1689,6 +1689,17 @@ private static bool GetIsWindows7Or2008R2()
             return false;
         }
 
+        internal void HeartBeat()
+        {
+            Http2Connection[]? localHttp2Connections = _http2Connections;
+            if (localHttp2Connections != null)
+            {
+                foreach (var http2Connection in localHttp2Connections)
```

Comment:
Do not use `var` when not immediately obvious from RHS.

```suggestion
                foreach (Http2Connection http2Connection in localHttp2Connections)
```